store/tikv/rawkv.go

```diff
@@ -131,6 +135,26 @@ func (c *RawKVClient) Put(key, value []byte) error {
 	return nil
 }
 
+// BatchPut stores key-value pairs to TiKV.
+func (c *RawKVClient) BatchPut(keys, values [][]byte) error {
+	start := time.Now()
+	defer func() {
+		metrics.TiKVRawkvCmdHistogram.WithLabelValues("batch_put").Observe(time.Since(start).Seconds())
+	}()
+
+	if len(keys) != len(values) {
+		return errors.New("the len of keys is not equal to the len of values")
```

Comment:
Any reason to check this? To me, key and value has different length is pretty normal. If you have a reason, maybe better add it as a comment.

Reply:
Maybe to say, the count of keys is not equal to the count of values.
The function receives an array of keys and an array of values, and they should match. So the two array's length should be equal.
I think maybe it's better to receive an array of key-value pairs as argument, instead of receiving two arrays

Reply:
The keys and values are the slice of `[]byte`.